shell/components/fleet/dashboard/ResourceCard.vue

```diff
@@ -0,0 +1,175 @@
+            tooltip:     {},
+            handleClick: () => {},
+          }];
+        }
+
+        return [];
+      }
+
+      return [{
+        id:          state.id,
+        icon:        state.icon,
+        color:       '',
+        customColor: state.color,
+        tooltip:     {},
+        handleClick: () => {},
+      }];
+    },
+
+    resourceCounts() {
+      const out = clone(this.resourcesDefaultStates);
+
+      const resourceStatuses = this.value.allResourceStatuses;
+
+      Object.entries(resourceStatuses.states)
+        .filter(([_, count]) => count > 0)
+        .forEach(([state, count]) => {
+          const k = state?.toLowerCase();
+
+          if (out[k]) {
+            out[k].count += count;
+          } else {
+            out.unknown.count += count;
+          }
+        });
+
+      return Object.values(out).reduce((acc, { label, count }) => {
+        if (count > 0) {
+          return [...acc, { label, count }];
+        }
+
+        return acc;
+      }, []).reverse();
+    },
+
+    resourcesTooltip() {
+      return this.resourceCounts.reduce((acc, state, i) => `${ acc }${ i > 0 ? '<br>' : '' }${ state.label }: ${ state.count }`, '');
+    },
+
+    noClusters() {
+      return !this.value.status?.desiredReadyClusters;
+    },
+
+    nameTooltip() {
+      if (this.value.nameDisplay?.length >= 15) {
+        return this.value.nameDisplay;
+      }
+
+      return null;
+    }
+  },
+
+  methods: {
+    select(value) {
+      const elem = value?.srcElement;
+
+      if (elem?.tagName === 'A' || elem?.tagName === 'BUTTON' || elem?.className.includes('icon icon-actions')) {
+        return;
+      }
+
+      this.$emit('click');
+    },
+  }
+};
+</script>
+
+<template>
+  <RcItemCard
+    :id="`${ value.metadata.namespace }-${ value.type }-${ value.id }`"
+    class="dashboard-resource-card"
+    variant="small"
+    :header="{
+      title: { text: value.nameDisplay },
+      image: {},
+      statuses,
+    }"
+    :content="{}"
+    :value="value"
+    @click="select"
+    @keydown.self.enter.stop.prevent="select"
+    @keydown.self.space.stop.prevent="$router.push(value.detailLocation)"
+  >
+    <template #item-card-image>
+      <i
+        class="icon-lg"
+        :class="value.dashboardIcon"
+      />
+    </template>
+    <template #item-card-actions>
+      <ActionMenu
+        :resource="value"
+        :button-aria-label="t('sortableTable.tableActionsLabel', { resource: value?.id || '' })"
+      />
+    </template>
+    <template #item-card-content>
+      <ResourceCardSummary
+        v-clean-tooltip="{content: resourcesTooltip, triggers: ['hover']}"
+        :value="value"
+        :no-clusters="noClusters"
+      />
+    </template>
+  </RcItemCard>
+</template>
+
+<style lang="scss" scoped>
+  .dashboard-resource-card {
+    margin: 1px;
+
+    :deep() .item-card-body {
+      width: -webkit-fill-available;
+    }
+  }
+
+  :deep() .item-card-header-title {
+    margin-left: 8px;
+  }
```

Comment:
nit: Ideally, we should only require deep selectors in unsupported edge cases. Consider syncing with @oboc-sts & @momesgin to see if there are any enhancements that can be made at this point to support all our card use-cases.

Reply:
Fixed by adding the margin to the icon